src/mds/nameserver2/curvefs.cpp

```diff
@@ -1751,6 +1770,11 @@ StatusCode CurveFS::RefreshSession(const std::string &fileName,
         return  ret;
     }
 
+    if (IsLock(context)) {
+        LOG(INFO) << "filename = " << fileName << "clientuuid = " <<
+            context->uuid() << " openflags = " << context->openflags();
+        writerlock_->UpdateLockTime(fileInfo->id(), context->uuid());
```

Comment:
ditto, and only LOG(WARNING) if failed, do nothing if succeeded, because refresh session is frequently than open and close.